deploy/charts/approver-policy/values.yaml

```diff
@@ -109,3 +109,6 @@ commonLabels: {}
 # -- Optional allow custom annotations to be placed on cert-manager-approver pod
 podAnnotations: {}
 
+crds:
+  # -- Whether or not to install the crds.
+  enabled: true
```

Comment:
Any reason for using a value different from the one used in cert-manager, ref. https://cert-manager.io/docs/installation/helm/#helm-installation?

Reply:
It's based on trust-manager: https://github.com/cert-manager/trust-manager/blob/main/deploy/charts/trust-manager/values.yaml#L184-L186

Reply:
Thanks, makes sense!